packages/vulcan-core/lib/modules/containers/withCreate.js

```diff
@@ -0,0 +1,59 @@
+/*
+
+Generic mutation wrapper to insert a new document in a collection and update
+a related query on the client with the new item and a new total item count. 
+
+Sample mutation: 
+
+  mutation updateMovie($document: MoviesInput) {
+    updateMovie(document: $document) {
+      ...MovieFormFragment
+    }
+  }
+
+Arguments: 
+
+  - document: the document to insert
+
+Child Props:
+
+  - createMovie(document)
+    
+*/
+
+import React, { Component } from 'react';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import { getFragment, getFragmentName, getCollection, createClientTemplate } from 'meteor/vulcan:core';
+
+export default function withCreate(options) {
```

Comment:
Functions are hoisted to the global scope and can hide circular dependencies. Load order should be controlled by the files themselves. By doing it as a variable function expression instead of a function declaration, you're creating a file-scoped variable that only exists when that line is loaded which prevents hoisting it outside of the file.

https://stackoverflow.com/a/39582542

Specifically: https://stackoverflow.com/a/336868